From 874596b1768a0e9fdc1919fd9b28be0575fd205f Mon Sep 17 00:00:00 2001
From: Andrea Adami <andrea.adami@gmail.com>
Date: Tue, 15 Jun 2021 00:17:39 +0200
Subject: [PATCH 1/2] tuxtxt-enigma2: fix warning and build with musl

fix
 | ../git/tuxtxt/tuxtxt.c:215:15: error: storage size of 's' isn't known
 |   215 |   struct stat s;
 |       |               ^

With glibc this is only:
 warning: implicit declaration of function 'stat' [-Wimplicit-function-declaration]

Signed-off-by: Andrea Adami <andrea.adami@gmail.com>
---
 tuxtxt.h | 1 +
 1 file changed, 1 insertion(+)

diff --git a/tuxtxt.h b/tuxtxt.h
index 7f40b09..c99ae5d 100644
--- a/tuxtxt.h
+++ b/tuxtxt.h
@@ -40,6 +40,7 @@
 #include <string.h>
 #include <unistd.h>
 #include <sys/types.h>
+#include <sys/stat.h>
 #include <sys/time.h>
 #include <ctype.h>
 
-- 
2.17.1

